common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -339,6 +348,12 @@ def test_member_of_a_team(self):
             self.my_teams_page.visit()
         self.verify_teams(self.my_teams_page, teams)
 
+    def test_multiple_team_members(self):
+        teams = self.create_teams(self.topic, 1)
+        self.create_memberships(4, teams[0]['id'])
```

Comment:
How does the active user get enrolled in the course? Through create_teams (in which case there should be 5 members, not 4)? Or somehow magically when creating 4 memberships via create_memberships?